fiftyone/core/annotation/validate_label_schemas.py

```
@@ -694,6 +696,25 @@ def _validate_values_setting(field_name, value, _type, key=foac.VALUES):
 
+def _validate_applied_ontology(field_name: str, value: str) -> None:
+    # Late import to avoid a circular import with the ontology SDK.
+    from fiftyone.core.ontology import load_ontology
+
+    try:
+        ontology = load_ontology(value)
+    except ValueError:
+        raise ValueError(
+            f"'{foac.APPLIED_ONTOLOGY}' references unknown ontology "
+            f"'{value}' for field '{field_name}'"
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)